llvm/test/tools/sycl-post-link/spec-constants/SYCL-alloca.ll

```diff
@@ -0,0 +1,60 @@
+; RUN: sycl-post-link -spec-const=native < %s -S -o %t.table
+; RUN: FileCheck %s -check-prefixes=CHECK-RT < %t_0.ll
+; RUN: FileCheck %s --check-prefixes=CHECK-PROPS < %t_0.prop
+
+; This test checks that the post link tool is able to correctly transform
+; SYCL alloca intrinsics in SPIR-V devices.
+
+%"class.sycl::_V1::specialization_id" = type { i64 }
+%"class.sycl::_V1::specialization_id.0" = type { i32 }
+%"class.sycl::_V1::specialization_id.1" = type { i16 }
+%my_range = type { ptr addrspace(4), ptr addrspace(4) }
+
+@size_i64 = internal addrspace(1) constant %"class.sycl::_V1::specialization_id" { i64 10 }, align 8
+@size_i32 = internal addrspace(1) constant %"class.sycl::_V1::specialization_id.0" { i32 120 }, align 4
+@size_i16 = internal addrspace(1) constant %"class.sycl::_V1::specialization_id.1" { i16 1 }, align 2
+
+; check that the following globals are preserved: even though they are won't be
```

Comment:
```suggestion
; Check that the following globals are preserved: even though they are won't be
```

Reply:
Also in `they are won't be` seems like either `are` or `won't be` should be gone.